uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -94,6 +94,162 @@ Feature: GGMQ-1
       | v5     | mosquitto-c | aws.greengrass.client.MqttMosquittoClient | client_mosquitto_c.yaml | GRANTED_QOS_1       |
 
 
+  @GGMQ-1-T2
+  Scenario Outline: GGMQ-1-T2-<mqtt-v>-<name>: GGAD can publish to an MQTT topic at QoS 0 and QoS 1 based on CDA configuration
```

Comment:
Do we need this to be a separate scenario? The tests are already taking over 1 hour to run. Combining scenarios will save significant time. For example, if we just had a single scenario which ran all the steps to validate what we want, this would be the fastest.